obkvrpc/connection_pool.go

```diff
@@ -139,11 +178,223 @@ func (p *ConnectionPool) CreateConnection(ctx context.Context) (*Connection, err
 	if err != nil {
 		return nil, errors.WithMessage(err, "connection login")
 	}
+	// put it to here to ensure connection should not expire during connect & login phase
+	if p.option.maxConnectionAge > 0 {
+		connection.expireTime = time.Now().Add(p.option.maxConnectionAge)
+	}
+	log.Info(fmt.Sprintf("connect success, remote addr:%s", connection.conn.RemoteAddr().String()))
 	return connection, nil
 }
 
 func (p *ConnectionPool) Close() {
 	for _, connection := range p.connections {
 		connection.Close()
 	}
+
+	if p.connMgr != nil {
+		p.connMgr.close()
```

Comment:
I am not sure what will happen when we close this Mgr without waiting for the coroutine stop

Reply:
done, in the latest version, the mgr will notify and wait the coroutine stop before close